### PR TITLE
I've made some changes.

### PR DESCRIPTION
**Pull request by hmotande-nizet**

### CODE REVIEW

File: questionfile.py [hunk 1]
```diff
@@ -19,3 +19,8 @@
 #50 legs
 
 #Create a python script to solve this problem.
+
```
Comment (NPCWill) [inline questionfile.py]: This will raise a value error if the user input is not an number

File: questionfile.py [hunk 2]
```diff
@@ -19,3 +19,8 @@
 #50 legs
 
 #Create a python script to solve this problem.
+
+chickens = int(input("how many chickens? "))
+cows = int(input("how many cows? "))
+dogs = int(input("how many dogs? "))
+legs = (chickens * 2) + (cows * 4) + (dogs * 4)
```
Comment (NPCWill) [inline questionfile.py]: This is good, but this will not give any feedback to the user. You should add a print statement.

**Review by NPCWill — CHANGES_REQUESTED** [review]

**Review by hmotande-nizet — COMMENTED** [review]
.

File: questionfile.py (hunk 1 above)
Comment (hmotande-nizet) [reply questionfile.py]: print("number of legs in total is {}".format(legs))

**Pull request merged**